src/array_api_stubs/_draft/_types.py

```diff
@@ -90,7 +88,7 @@ def __len__(self, /) -> int:
         ...
 
 
-class Info(Protocol):
+class Info(Protocol[device]):
```

Comment:
Can you explain what the `[device]` here means after Protocol? The Info namespace object itself does not depend on the device (`__array_namespace_info__` does not take a `device` parameter).